README.rst

```diff
@@ -104,10 +103,8 @@ Click and drag to pan around.
 `View a demo of the LMS`_
 
 
-.. |badge-coveralls| image:: https://coveralls.io/repos/github/Stanford-Online/xblock-image-modal/badge.svg?branch=master
-   :target: https://coveralls.io/github/Stanford-Online/xblock-image-modal?branch=master
```

Comment:
This doesn't work if you s/Stanford-Online/openedx/ -- I think because it is a paid service (that Stanford is still paying for...??)